src/style.css

```diff
@@ -0,0 +1,59 @@
+html {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.bar {
+  margin: 20px 8px;
+}
+
+.search {
+  width: 100%;
+  height: 70px;
+  padding: 5px 62px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 50%;
+  background-size: 20px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  line-height: 22px;
+  font-size: 16px;
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+}
+
+.small-class {
+  font-size: 14px;
+  height: 42px;
+  padding: 5px 33px;
+  background-position: 12px 48%;
+  background-size: 11px;
+}
+
+.search:hover {
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
`linear-gradient` wouldn't work with `background-color`.
You should use `background` shorthand, but consider specifics of it's syntax.
Basically you'd have to put both gradient and icon image inside and state alignment and object-fit.
Read about it here: https://developer.mozilla.org/en-US/docs/Web/CSS/background
this should give you the idea:
![Snapshot](https://user-images.githubusercontent.com/39308717/159009072-954ad235-5468-4991-90cc-680b94acd1d8.png)

you'll need to also position and size your background (`background-position / -size`)